VimR/VRFileBrowserOutlineView.m

```diff
@@ -195,6 +205,35 @@ - (BOOL)processKeyModeNormal:(unichar)key {
       _actionMode = VRFileBrowserActionModeMenu;
       [self.actionDelegate updateStatusMessage:@"Actions: (a)dd (m)ove (d)elete (c)opy"];
       return YES;
+    case 'g':
+      if (_lastKey == 'g') {
+        _lastKey = cZero;
+        [self.actionDelegate actionMoveToTop];
+      }
+      return YES;
+    case 'G':
+      [self.actionDelegate actionMoveToBottom];
+      return YES;
+    case 'e':
+      if (modifierFlags & NSControlKeyMask) {
+        [self.actionDelegate actionScrollDownOneLine];
+      }
+      return YES;
+    case 'y':
+      if (modifierFlags & NSControlKeyMask) {
+        [self.actionDelegate actionScrollUpOneLine];
+      }
+      return YES;
+    case 'f':
+      if (modifierFlags & NSControlKeyMask) {
+        [self.actionDelegate actionScrollDownOneScreen];
+      }
+      return YES;
+    case 'b':
+      if (modifierFlags & NSControlKeyMask) {
+        [self.actionDelegate actionScrollUpOneScreen];
+      }
+      return YES;
```

Comment:
For `'e'`, `'y'`, `'f'`, and `'b'` `YES` is returned even if the control key isn't pressed. I think `return YES` should be moved into the `if` statements and `NO` should be returned otherwise.